scripts/send-email-to-unresolved-breach-subscribers.js

```diff
@@ -2,18 +2,28 @@ const DB = require('../db/DB')
 const EmailUtils = require('../email-utils')
 const { LocaleUtils } = require('../locale-utils')
 const AppConstants = require('../app-constants')
-const { argv } = require('node:process')
+const { env, argv } = require('node:process')
 
-async function sendUnresolvedBreachEmails (subscribers = null) {
-  let count = 0
-
-  LocaleUtils.init()
-  await EmailUtils.init()
-  await DB.createConnection()
-
-  if (!subscribers) subscribers = await DB.getSubscribersWithUnresolvedBreaches() // if test subscribers are not available, pull from db
+/* Send a monthly email to each subscriber with unresolved breaches
+ *
+ * Usage:
+ * node scripts/send-email-to-unresolved-breach-subscribers.js
+ *
+ * For testing, pass a comma-separated arg with no spaces, for example:
+ * node scripts/send-email-to-unresolved-breach-subscribers.js test1@test.com,test2@test.com,...
+ */
 
-  console.log('- Recipient total:', subscribers.length)
+async function sendUnresolvedBreachEmails (subscribers = null, limit = 0) {
+  let count = 0
+  let total = 0
+  if (subscribers) {
+    total = subscribers.length
+  } else {
+    // if test subscribers are not available, pull from db
+    total = await DB.getSubscribersWithUnresolvedBreachesCount()
+    subscribers = await DB.getSubscribersWithUnresolvedBreaches(limit)
```

Comment:
If the limit (100)  less than the total count (9061):

```
Running job for monthly unresolved breach emails...
- Attempting to email 100 of 100 subscribers...
- Successfully emailed 100 of 100 subscribers (9061 total)
```

And if the limit (100) is greater than the total count (37):

```
Running job for monthly unresolved breach emails...
- Attempting to email 37 of 37 subscribers...
- Successfully emailed 37 of 37 subscribers (37 total)
```

I think this is interesting data - we know how many subscribers are due for a monthly email, versus how many we sent. I'm not very worried about the cost of the extra query, since it is a background task. There is more processing load on the database, but returning a count is a lot less data than that many rows of subscribers.

Reply:
Ah, I didn't see this line before, and thought `subscribers.length` was being neglected:
https://github.com/mozilla/blurts-server/blob/8b927d019b3f816191c341cdb3dd022f760933e0/scripts/send-email-to-unresolved-breach-subscribers.js#L51

Agreed that the total context is helpful 👍